tools/astarte_e2e/lib/astarte_e2e/service_notifier/email.ex

```diff
@@ -22,21 +22,61 @@ defmodule AstarteE2E.ServiceNotifier.Email do
   alias AstarteE2E.Config
 
   def service_down_email(reason) do
-    from = Config.mail_from_address!()
-    to = Config.mail_to_address!()
-
     text = """
     AstarteE2E detected a service malfunction.
 
     Reason: #{reason}.
+    #{datetime_to_string()}
 
     Please take actions to prevent further issues.
     """
 
+    base_email()
+    |> subject("Astarte Warning! Service is down")
+    |> text_body(text)
+  end
+
+  def service_up_email do
+    text = """
+    AstarteE2E service is back to its normal state.
+
+    #{datetime_to_string()}
+    """
+
+    base_email()
+    |> subject("Astarte: service back to its normal state.")
+    |> text_body(text)
+  end
+
+  defp base_email do
+    from = Config.mail_from_address!()
+    to = Config.mail_to_address!()
+
     new_email()
     |> to(to)
     |> from(from)
-    |> subject("Astarte Warning! Service is down")
-    |> text_body(text)
+  end
+
+  defp datetime_to_string do
```

Comment:
Switch this simply to 
```
def current_timestamp do
  DateTime.utc_now() |> DateTime.to_string()
end
```
An ISO 8601 timestamp is more handy to handle (e.g. it can be directly copypasted in the `--since-time` option of `kubectl logs` and so on)